bedrock/base/templates/macros.html

```diff
@@ -234,3 +234,55 @@ <h2 class="mzp-c-form-title">{{ form_title }}</h2>
     <img src="{{ src }}" alt="{{ alt }}" {% if class %}class="{{ class }}" {% endif %}{% if loading %}loading="{{ loading }}" {% endif %}{% if width %}width="{{ width }}" {% endif %}{% if height %}height="{{ height }}" {% endif %}/>
   {% endif %}
 {%- endmacro %}
+
+{% macro sub_nav(
+  title,
+  links
+) -%}
+
+{% if 'releasenotes' in request.path %}
```

Comment:
Yeah, this is an odd one.

tl;dr: the if statement is here because there's a discrepancy between the request path and the link hrefs in release note pages and it felt simpler to manage here than in the note.html sub_nav call. 

request.path = /en-US/firefox/98.0.2/releasenotes/
link.href = /en-US/firefox/notes/

request.path = /en-US/firefox/99.0beta/releasenotes/
link.href = /en-US/firefox/beta/notes/

More detail:

I had no active page link styling on release note sub nav because link.href never === request.path. To get that active page link styling, I saw a few options:

1. Create a `current_page` variable in the macro and set that conditionally if we're on one of the release pages. Default to the `request.path` value.
2. Create a `current_page` param for the macro and set it conditionally in the `notes.html`template before making the sub_nav call. Param default is request.path.
3. Change the link hrefs in the notes sub_nav call to match the request path structure
4. Change the request path in the notes sub_nav call to match the link href structure

3 didn't scale well and had the potential to break links.
4 seemed like bad practice, with potential unintended consequences.

2 is still viable, I think, although I have to check if you can use context for a param default (the if statement might have to stay, i.e. if current_page is set, use that, else, use request.path). It does involve creating a new param, because otherwise there is no way to pass the matchable request path to the macro.

1 seemed simplest, as it didn't involve creating a new param that mostly wouldn't be used and kept the logic in one place that could handle more conditions if needed. BUT, if it causes confusion, maybe it's worth taking another look at option 2. Let me know what sounds best to you!

Reply:
@reemhamz I'm going to go ahead and merge as-is for now, but we can definitely open a new, more focused issue to deal with the release page weirdness if it seems worthwhile